### PR TITLE
Add logging with Azure.Monitor.OpenTelemetry.AspNetCore section

### PR DESCRIPTION
**Pull request by lmolkova**
Updates Azure SDK .NET logging article according to https://github.com/Azure/azure-sdk-for-net/pull/42374

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/azure/sdk/logging.md](https://github.com/dotnet/docs/blob/c29284cbcad3cbe70c50e51bae28a5a3a54faf60/docs/azure/sdk/logging.md) | [Logging with the Azure SDK for .NET](https://review.learn.microsoft.com/en-us/dotnet/azure/sdk/logging?branch=pr-en-us-39854) |

### CODE REVIEW

File: docs/azure/sdk/logging.md [hunk 1]
```diff
@@ -186,6 +186,31 @@ In these scenarios, complete the following steps:
...
 For more information, see [Logging in .NET Core and ASP.NET Core](/aspnet/core/fundamentals/logging/).
 
+## Logging using Azure.Monitor.OpenTelemetry.AspNetCore
+
+[Azure Monitor OpenTelemetry Distro](https://www.nuget.org/packages/Azure.Monitor.OpenTelemetry.AspNetCore) starting with version `1.2.0` supports capturing logs coming from Azure client libraris. 
+You can control logging using any of the [Logging in .NET Core and ASP.NET Core configuration options](/aspnet/core/fundamentals/logging/). 
+
+Using the Azure Service Bus library as an example, complete the following steps:
+
+1. Install the [/Azure.Monitor.OpenTelemetry.AspNetCore](https://www.nuget.org/packages/Azure.Monitor.OpenTelemetry.AspNetCore) NuGet package:
```
Comment (lmolkova) [inline docs/azure/sdk/logging.md:196]:
```suggestion
1. Install the [Azure.Monitor.OpenTelemetry.AspNetCore](https://www.nuget.org/packages/Azure.Monitor.OpenTelemetry.AspNetCore) NuGet package:
```

File: docs/azure/sdk/logging.md [hunk 2]
```diff
@@ -186,6 +186,31 @@ In these scenarios, complete the following steps:
 
 For more information, see [Logging in .NET Core and ASP.NET Core](/aspnet/core/fundamentals/logging/).
 
+## Logging using Azure.Monitor.OpenTelemetry.AspNetCore
+
+[Azure Monitor OpenTelemetry Distro](https://www.nuget.org/packages/Azure.Monitor.OpenTelemetry.AspNetCore) starting with version `1.2.0` supports capturing logs coming from Azure client libraris.
```
Comment (jsquire) [inline docs/azure/sdk/logging.md:191]:
```suggestion
[Azure Monitor OpenTelemetry Distro](https://www.nuget.org/packages/Azure.Monitor.OpenTelemetry.AspNetCore),starting with version `1.2.0`, supports capturing logs coming from Azure client libraries. 
```

File: docs/azure/sdk/logging.md [hunk 3]
```diff
@@ -186,6 +186,31 @@ In these scenarios, complete the following steps:
 
 For more information, see [Logging in .NET Core and ASP.NET Core](/aspnet/core/fundamentals/logging/).
 
+## Logging using Azure.Monitor.OpenTelemetry.AspNetCore
+
+[Azure Monitor OpenTelemetry Distro](https://www.nuget.org/packages/Azure.Monitor.OpenTelemetry.AspNetCore) starting with version `1.2.0` supports capturing logs coming from Azure client libraris. 
+You can control logging using any of the [Logging in .NET Core and ASP.NET Core configuration options](/aspnet/core/fundamentals/logging/).
```
Comment (jsquire) [inline docs/azure/sdk/logging.md:192]:
```suggestion
You can control logging using any of the configuration options discussed in [Logging in .NET Core and ASP.NET Core](/aspnet/core/fundamentals/logging/).
```

File: docs/azure/sdk/logging.md [hunk 4]
```diff
@@ -186,6 +186,31 @@ In these scenarios, complete the following steps:
...
+    ```dotnetcli
+    dotnet add package Azure.Monitor.OpenTelemetry.AspNetCore
+    ```
+
+1. Create or register the Azure SDK library's client - the distro supports both cases
+
+   ```csharp
+   await using var client = new ServiceBusClient("<connection_string>");
+   ```
+
```
Comment (jsquire) [inline docs/azure/sdk/logging.md:207]:
```suggestion

```

**Review by jsquire — APPROVED** [review]

File: docs/azure/sdk/logging.md [hunk 5]
```diff
@@ -186,6 +186,31 @@ In these scenarios, complete the following steps:
 
 For more information, see [Logging in .NET Core and ASP.NET Core](/aspnet/core/fundamentals/logging/).
 
+## Logging using Azure.Monitor.OpenTelemetry.AspNetCore
+
+[Azure Monitor OpenTelemetry Distro](https://www.nuget.org/packages/Azure.Monitor.OpenTelemetry.AspNetCore),starting with version `1.2.0`, supports capturing logs coming from Azure client libraries.
```
Comment (christothes) [inline docs/azure/sdk/logging.md:191]:
```suggestion
[Azure Monitor OpenTelemetry Distro](https://www.nuget.org/packages/Azure.Monitor.OpenTelemetry.AspNetCore), starting with version `1.2.0`, supports capturing logs coming from Azure client libraries.
```

**Review by JoshLove-msft — APPROVED** [review]

**Review by christothes — APPROVED** [review]

**Comment by lmolkova** [comment]
@gewarren @BillWagner is there something blocking this PR from being merged?

File: docs/azure/sdk/logging.md [hunk 6]
```diff
@@ -186,6 +186,31 @@ In these scenarios, complete the following steps:
...
+
+Using the Azure Service Bus library as an example, complete the following steps:
+
+1. Install the [Azure.Monitor.OpenTelemetry.AspNetCore](https://www.nuget.org/packages/Azure.Monitor.OpenTelemetry.AspNetCore) NuGet package:
+
+    ```dotnetcli
+    dotnet add package Azure.Monitor.OpenTelemetry.AspNetCore
+    ```
+
+1. Create or register the Azure SDK library's client - the distro supports both cases
```
Comment (gewarren) [inline docs/azure/sdk/logging.md:202]:
```suggestion
1. Create or register the Azure SDK library's client - the distro supports both cases.
```

**Review by gewarren — APPROVED** [review]
LGTM. Feel free to squash and merge when you're ready. It looks like neither Bill nor I were tagged for review which is why we missed it.

**Comment by lmolkova** [comment]
@gewarren thanks a lot for taking a look, I don't have permissions to merge. Could you please hit the button? Thanks!

**Pull request merged**